Source/Controllers/SubviewControllers/SPUserManager.m

```diff
@@ -510,13 +507,8 @@ - (NSManagedObjectContext *)managedObjectContext
 
 - (void)beginSheetModalForWindow:(NSWindow *)docWindow completionHandler:(void (^)(void))callback
 {
-	//copy block from stack to heap, otherwise it wouldn't live long enough to be invoked later.
-	void *heapCallback = callback? Block_copy(callback) : NULL;
-
 	[docWindow beginSheet:self.window completionHandler:^(NSModalResponse returnCode) {
-		//directly invoking callback would risk that we are dealloc'd while still in this run loop iteration.
-		dispatch_async(dispatch_get_main_queue(), heapCallback);
-		Block_release(heapCallback);
```

Comment:
Is this safe now? Cause I see no longer async

Reply:
Yes, the comment said that we would be deallocated, but we don't care now, so we don't need to switch to async later.